tests/baselines/reference/asyncImportedPromise_es6.js

```diff
@@ -21,7 +21,7 @@ var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, ge
         function fulfilled(value) { try { step(generator.next(value)); } catch (e) { reject(e); } }
         function rejected(value) { try { step(generator.throw(value)); } catch (e) { reject(e); } }
         function step(result) { result.done ? resolve(result.value) : new P(function (resolve) { resolve(result.value); }).then(fulfilled, rejected); }
-        step((generator = generator.call(thisArg, _arguments)).next());
+        step((generator = generator.apply(thisArg, _arguments)).next());

```

Comment:
Before, I would rewrite `arguments` to `_arguments` inside of the generator function that serves as the body of the async function. I found it was simpler to remove that rewrite and instead call `apply`. In doing that, the `arguments` object is essentially correct, and the emit is less convoluted.